src/ui/Facet/FacetSettings.ts

```diff
@@ -320,7 +340,12 @@ export class FacetSettings extends FacetSort {
 
     saveStateSection.appendChild(icon);
     saveStateSection.appendChild(saveStateItems);
-    $$(saveStateSection).on('click', (e: Event) => this.handleSaveStateClick());
+    new AccessibleButton()
+      .withElement(saveStateSection)
+      .withSelectAction(() => this.handleSaveStateClick())
+      .withLabel(l('SaveFacetState'))
```

Comment:
I think this is closer to programmer terminology. I don't know what the translation is, but could `Save Facet Settings` be a better fit?

Similarly, `Reset Facet Settings` instead of `ClearFacetState`

Reply:
It's something to save the "current state of the facet" (selection/exclusion), and not "settings".

So the terminology seems okay to me.